### DISCUSSION TITLE
Failure to start latest image on Synology

### DISCUSSION THREAD

**Discussion by throttlemeister** [dani-garcia/vaultwarden#3000] [Q&A]
I am getting the following error while trying to start the container using the latest image:

> Start container Vaultwarden failed: {"message":"OCI runtime create failed: container_linux.go:367: starting container process caused: exec: \"/usr/bin/dumb-init\": stat /usr/bin/dumb-init: no such file or directory: unknown"}.

Reverting back to the previously used image works just fine and the container starts and application works.

I have tried deleting the image and downloading it again, but same result. I am running on Synology, no LXC containers or other virtualization present.

**Comment by BlackDex** [comment]
See:

- #2826 
- #2863 
- #2854
